examples/js/loaders/GLTFLoader.js

```diff
@@ -3122,11 +3122,11 @@ THREE.GLTFLoader = ( function () {
 
 				// Ambient lighting, if present, is always attached to the scene root.
 				if ( sceneDef.extensions
-						 && sceneDef.extensions[ EXTENSIONS.KHR_LIGHTS ]
-						 && sceneDef.extensions[ EXTENSIONS.KHR_LIGHTS ].light !== undefined ) {
+						 && sceneDef.extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ]
+						 && sceneDef.extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ].light !== undefined ) {
 
-					var lights = extensions[ EXTENSIONS.KHR_LIGHTS ].lights;
-					scene.add( lights[ sceneDef.extensions[ EXTENSIONS.KHR_LIGHTS ].light ] );
+					var lights = extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ].lights;
+					scene.add( lights[ sceneDef.extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ].light ] );
 
 				}
```

Comment:
This whole section can be removed, since ambient lights are no longer in the spec and were the only type attached to the scene.